src/builder/create_interaction_response.rs

```diff
@@ -64,6 +64,13 @@ pub enum CreateInteractionResponse {
     /// Corresponds to Discord's `PREMIUM_REQUIRED'.
     #[deprecated = "use premium button components via `CreateButton::new_premium` instead"]
     PremiumRequired,
+    /// Not valid for autocomplete and Ping interactions. Only available for applications with
+    /// Activities enabled.
+    ///
+    /// Responds to the interaction by launching the Activity associated with the app.
```

Comment:
Can these lines be flipped? Seems to be backwards.

Reply:
This mirrors the docs for the other variants, like `Acknowedge`, `UpdateMessage`, `AutoComplete`, and `Modal`. If this should be changed, I think the others should too.

Reply:
Ah, no worries then.